### PR TITLE
Add user structure

### PR DESCRIPTION
**Pull request by mplekunov**

### CODE REVIEW

File: server/serverAPI/model/weight/WeightUnits.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/**
```
Comment (nashirj) [inline server/serverAPI/model/weight/WeightUnits.ts:1]: Did you get these units from Spoonacular's API?

File: server/serverAPI/model/weight/WeightSchema.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    @IsNumber()
+    @IsPositive()
+    readonly value: number;
+
+    constructor(
+        unit: WeightUnits,
+        value: number
+    ) {
+        this.unit = unit;
+        this.value = value;
```
Comment (nashirj) [inline server/serverAPI/model/weight/WeightSchema.ts:22]: What happens if a unit is passed that's not in the `WeightUnits`? Maybe we can add something for `Undefined` weight?

File: server/serverAPI/model/user/UserSchema.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    firstName: string;
+    
+    @IsNotEmpty() 
+    lastName: string;
+    
+    @IsNotEmpty()
+    uid: string;
+
+    @IsInt()
+    lastSeen: number;
```
Comment (nashirj) [inline server/serverAPI/model/user/UserSchema.ts:18]: What are the units of `lastSeen`?

File: server/serverAPI/model/nutrients/INutrient.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { IsArray, IsInt, IsNotEmpty, IsObject, IsPositive, IsString } from "class-validator";
+
+/**
+ * Nutrient interface.
+ */
+export default interface INutrient {
+    readonly name: string;
+    readonly value: number;
```
Comment (nashirj) [inline server/serverAPI/model/nutrients/INutrient.ts:8]: What does `number` represent?

File: server/serverAPI/model/nutrients/INutrient.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import { IsArray, IsInt, IsNotEmpty, IsObject, IsPositive, IsString } from "class-validator";
+
+/**
+ * Nutrient interface.
+ */
+export default interface INutrient {
```
Comment (nashirj) [inline server/serverAPI/model/nutrients/INutrient.ts:6]: Could we add percent of daily recommended value?

File: server/Server.ts [hunk 6]
```diff
@@ -0,0 +1,4 @@
+import { server } from "./App";
+
+// Starts Server at specified Port
+server(5000);
```
[the remark this answers is not in the document]
Comment (mplekunov) [reply server/Server.ts:4]: for testing purposes... When u mock stuff like this, I have to have them separate. Besides it's also a pretty known practice. People have stuff in App and then some configs in Index.js in my case I named it Server.js for convinience.

**Pull request closed without merging**